Mathlib/MeasureTheory/Covering/VitaliFamily.lean

```diff
@@ -206,7 +206,7 @@ def enlarge (v : VitaliFamily μ) (δ : ℝ) (δpos : 0 < δ) : VitaliFamily μ
     have : ∀ x ∈ s, ∀ ε : ℝ, ε > 0 → ∃ (a : Set α), a ∈ g x ∧ a ⊆ closedBall x ε := by
       intro x hx ε εpos
       obtain ⟨a, af, ha⟩ : ∃ a ∈ f x, a ⊆ closedBall x (min ε δ)
-      exact ffine x hx (min ε δ) (lt_min εpos δpos)
+      · exact ffine x hx (min ε δ) (lt_min εpos δpos)
```

Comment:
Can you use the `obtain ... := by exact ...` syntax here? (This instance was missed in #11045.)
Or would you prefer me filing a PR for this and few similar instances, which can land before this one?

Reply:
Done!